src/components/sections/Hero.astro

```diff
@@ -12,21 +12,21 @@ import Container from "../shared/Container.astro";
         <span class="w-4/12 lg:w-2/12 aspect-square bg-gradient-to-tr from-primary to-green-400 absolute -top-5 lg:left-0 rounded-full skew-y-12 blur-2xl opacity-40 skew-x-12 rotate-90"></span>
         <div class="relative flex flex-col items-center text-center lg:text-left lg:py-7 xl:py-8 
         lg:items-start lg:max-w-none max-w-3xl mx-auto lg:mx-0 lg:flex-1 lg:w-1/2">
-            
+          
             <h1 class="text-3xl/tight sm:text-4xl/tight md:text-5xl/tight xl:text-6xl/tight
              font-bold text-heading-1">
              We provide <span class="text-transparent bg-clip-text bg-gradient-to-br from-indigo-600 from-20% via-primary via-30% to-green-600">prosthetics</span> to those who cannot afford them
             </h1>
             <div class="mt-10  w-full flex max-w-md mx-auto lg:mx-0">
                 <div class="flex sm:flex-row flex-col gap-5 w-full">
-                  <form action="https://formspree.io/f/mdojkeww" method="POST" id="my-form" class="py-1 pl-6 w-full pr-1 flex gap-3 items-center text-heading-3 shadow-lg shadow-box-shadow border border-box-border bg-box-bg rounded-full ease-linear focus-within:bg-body focus-within:border-primary">
+                  <form action="https://formspree.io/f/xaygwjrz" method="POST" id="my-form" class="py-1 pl-6 w-full pr-1 flex gap-3 items-center text-heading-3 shadow-lg shadow-box-shadow border border-box-border bg-box-bg rounded-full ease-linear focus-within:bg-body focus-within:border-primary">
                     <span class="min-w-max pr-2 border-r border-box-border">
                       <svg xmlns="http://www.w3.org/2000/svg" width="20" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="w-5 h-5">
                         <path stroke-linecap="round" stroke-linejoin="round" d="M21.75 9v.906a2.25 2.25 0 01-1.183 1.981l-6.478 3.488M2.25 9v.906a2.25 2.25 0 001.183 1.981l6.478 3.488m8.839 2.51l-4.66-2.51m0 0l-1.023-.55a2.25 2.25 0 00-2.134 0l-1.022.55m0 0l-4.661 2.51m16.5 1.615a2.25 2.25 0 01-2.25 2.25h-15a2.25 2.25 0 01-2.25-2.25V8.844a2.25 2.25 0 011.183-1.98l7.5-4.04a2.25 2.25 0 012.134 0l7.5 4.04a2.25 2.25 0 011.183 1.98V19.5z" />
                       </svg>                                                                  
                     </span>
                     <input type="email" name="email" id="" placeholder="johndoe@gmail.com" class="w-full py-3 outline-none bg-transparent">
-                    <button type="submit" class="min-w-max text-white bg-primary hover:bg-primary-dark rounded-full px-6 py-3" id="submit-btn">
+                    <Button variant={"primary"} className={"min-w-max text-white"}>
```

Comment:
add class "bg-primary hover:bg-primary-dark rounded-full px-6 py-3" id="submit-btn" to <Button> 

#2